### PR TITLE
Added Aditi Ahuja commuinty member  profile

### LINKED ISSUE #1732: [Community] Member Profile: Aditi Ahuja
**Current Behavior**

@metonymic-smokey  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @metonymic-smokey  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: metonymic-smokey
- Twitter: Aditi09039216
- LinkedIn: aditi-ahuja-334394193
- Link to profile picture: https://drive.google.com/file/d/1sROf4zwuDpy-4u44wvSD-1BstkEzojT5/view?usp=sharing

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Abiola-Farounbi**
Signed-off-by: abiola farounbi <farounbiabiola@gmail.com>

**Description**

This PR fixes #1732 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Abiola-Farounbi is the site building for you?

**Comment by Abiola-Farounbi** [comment]
No, it didn't

File: src/collections/members/aditi-ahuja/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Aditi Ahuja
+position: Contributor
+image_path: ./aditi-ahuja.jpg
+github: metonymic-smokey
+twitter: Aditi09039216
+linkedin: aditi-ahuja-334394193
+location: Bangalore, India
+bio: Aditi Ahuja is a Student developer from Bangalore, India passionate about venturing into newer, niche fields.
+Open to challenges and opportunities either technical or otherwise
```
Comment (Nikhil-Ladha) [inline src/collections/members/aditi-ahuja/index.mdx:10]: Try adding a space before this line, and build. See if that works.

Comment (Abiola-Farounbi) [reply src/collections/members/aditi-ahuja/index.mdx:10]: It didn't work

Comment (Nikhil-Ladha) [reply src/collections/members/aditi-ahuja/index.mdx:10]: This should be one line with spaces, not separate lines. And, don't add an empty line after it.

**Pull request closed without merging**